eth2/state_processing/src/per_block_processing/block_signature_verifier.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::integer_arithmetic)]
```

Comment:
Why do we allow it here? I can't see any arithmetic in here, perhaps it recurses into called functions?

Reply:
It's the div here: https://github.com/sigp/lighthouse/blob/3e4dc7ac4ee05fba2d47b6aaa2f6fbd65324d447/eth2/state_processing/src/per_block_processing/block_signature_verifier.rs#L130

Reply:
Watch out in case your machine has 0 cores ( ͡° ͜ʖ ͡°)